boards/nucleo-wl55jc/Makefile.dep

```diff
@@ -1,4 +1,8 @@
 ifneq (,$(filter stdio_uart,$(USEMODULE)))
   FEATURES_REQUIRED += periph_lpuart
 endif
+ifneq (,$(filter netdev_default,$(USEMODULE)))
+  USEMODULE += sx1261
```

Comment:
as discussed offline, this should only select the cpu specific version (`sx126x_stm32`)

Reply:
Done